install_sct

```diff
@@ -537,6 +537,13 @@ if [[ $UID != 0 ]]; then
   if [[ ${OS} == "osx" ]]; then
     sed -ie '/sct_env/ s/^#*/#/' ${THE_RC}
   else
+    # For Ubuntu-14.04 we need to export a path to make sure the build does not fail with pytorch
+    if uname -a | grep 14.04; then
+        sudo apt-get install libtcmalloc-minimal4
+        echo "" >> ${THE_RC}
+        echo "export LD_PRELOAD='/usr/lib/libtcmalloc_minimal.so.4'" >> ${THE_RC}
```

Comment:
This seems to have failed, maybe I am not exporting it into the right bash profile ? I noticed there are other adresses like `${THE_BASHRC}` but I am not sure if I placed it correctly. Any thoughts about this @jcohenadad or @charleygros ? Besides it is weird because the built fails right at the end after having successfully tested all functions...

Reply:
@bsauty can you update the bash profile [here](https://github.com/neuropoly/spinalcordtoolbox/blob/master/install_sct#L91) (to cover both bash and t/csh)